Makefile

```diff
@@ -14,7 +16,10 @@ install:
 install-dependencies:
 	go get -u github.com/stretchr/testify/...
 install-tools:
+
 test:
 	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s -race ./...
+test-full:
+	$(ROOT_DIR)/scripts/test-full.sh
 test-verbose:
```

Comment:
"I would however do all tests in test-full with the verbose flag, since we would run that in the CI." <- if you as well do that then both will do the same and I don't see the merit of keeping them both